accessing
error

	^(self at: #error)